src/include/migraphx/value.hpp

```diff
@@ -370,11 +378,11 @@ struct value
     }
 
     template <class To>
-    To value_or(const To& default_value) const
+    literal_to_string<To> value_or(const To& default_value) const
```

Comment:
Unrelated but on line 360 in value.hpp inside `visit_value()`,  `MIGRAPHX_VALUE_GENERATE_CASE_VALUE` is unused in that method.  Do we need those cases to be generated as well ?

Reply:
Its unused because I pass `MIGRAPHX_VALUE_GENERATE_CASE` instead of ` MIGRAPHX_VALUE_GENERATE_CASE_VALUE`? That is definitely a copy-paste mistake that should be fixed.